code/modules/sanity/breakdown.dm

```diff
@@ -49,6 +49,11 @@
 	flick(icon_state, img)
 
 /datum/breakdown/proc/occur()
+	if(!holder)
+		return FALSE
+	if(!holder.owner)
+		return FALSE
+
```

Comment:
Again, we might never know why in the Hell breakdown is trying to occur where there is no sanity or where there's no mob who should hold this sanity datum. Just hides away the problem

Reply:
If a mob receives breakdown w/o sanity, it's probably an admincall, rather than actual bug. Since holder is sanity and owner is a mob. If sanity exists w/o mob, then... i dont really know why it exists then.

Reply:
I'm pretty sure there was no case of manual admincalling breakdowns of people in a long time already. Moreover why would admin call a breakdown on a deleted mob?
Runtimes should be used to trace the reason why this even happens and without those runtimes we might end up with improper deletion of mobs leaving sanity module behind and still processing or some other obscure bug.
Leaving this error unattended and silencing it with a check might just fuck things up even further!